.github/workflows/pack_patches.yml

```
@@ -0,0 +1,30 @@
+name: Pack Patches
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
```

`branches: *` does nothing.
```suggestion
on: [push, pull_request]
```